ios/Podfile.lock

```diff
@@ -665,11 +670,12 @@ SPEC CHECKSUMS:
   RNFBAnalytics: 2dc4dd9e2445faffca041b10447a23a71dcdabf8
   RNFBApp: 7eacc7da7ab19f96c05e434017d44a9f09410da8
   RNFBCrashlytics: 4870c14cf8833053b6b5648911abefe1923854d2
+  RNShareMenu: 555df30344da0d60947ccc781b0447ee9e5cd48f
   RNSVG: ce9d996113475209013317e48b05c21ee988d42e
   urbanairship-react-native: dfb6dc22b2f41ccaadd636b73d51b448cd1b2bbc
   Yoga: 7d13633d129fd179e01b8953d38d47be90db185a
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: 41b806c7f131f87b716be1f1f9377532d6c9e43a
+PODFILE CHECKSUM: dc731cc579bb3ba93e3ed267d05524db4ca67bc6
 
-COCOAPODS: 1.10.0
+COCOAPODS: 1.10.1
```

Comment:
Would you mind reverting to 1.10.0 to build this project? We haven't switched to 1.10.1 quite yet and it would help keep our mobile repos in sync.

I managed to do it by reinstalling the cocoapods gem to that version and deleting this file before a pod install.

Reply:
I had to use 1.10.1, because you have this version in your Gemfile.lock

https://github.com/Expensify/Expensify.cash/blob/d353a46e95efd3e9d0df85134e078aa0e859b8fa/Gemfile.lock#L34

I had 1.10.0 too, but `pod install` wasn't working and was showing following error: `Could not find proper version of cocoapods (1.10.1) in any of the sources`.

So I'm not sure how to proceed. I can revert to 1.10.0, but are you sure that it will work?

Reply:
Ah I see, I didn't realize it was locked to that version in the gemfile. Let me double check on that.